src/main/java/frc/robot/Robot.java

```diff
@@ -684,7 +684,7 @@ private void executeClimbingStateMachine() {
         mClimber.retract();
 
         /** Checks if the motor is overcurrenting, signalling it's done retracting */
-        if (mClimber.checkOvercurrent()) {
+        if (mClimber.readyForNextState()) {
           mClimbingState = ClimbingState.RETRACTING_COMPLETE;
         }
       case RETRACTING_COMPLETE:
```

Comment:
This whole state is redundant, since the IDLE state stops the climber.

Reply:
Deleted